templates/hello/simple-snippet.html

```diff
@@ -0,0 +1,237 @@
+<!-- Any instances of {{ snippet_id }} are replaced with the ID number for the
+     snippet using this template. -->
+
+<div class="snippet" id="{{ snippet_id }}-hello-simple">
+    <style>
+      #{{ snippet_id }}-ffos-circling-fox {
+          /* All CSS should be preceeded by the ID selector for this individual
+            snippet. */
+      }
+    </style>
+    
+    {% if icon_url %}<a href="#" id="hello-cta">{% endif %}
+    <img class="icon" src="{{ icon }}" />
+    {% if icon_url %}</a>{% endif %}
+    <p>{{ text|safe }}</p>
+
+    <script type="text/javascript">
+      //<![CDATA[
+      var TARGET_AVAILABLE;
+      var TARGET_NAME = 'loop';
+      var INELIGIBLE_VIEW = 'hello-ineligible-view';
+      var INELIGIBLE_CLICK = 'hello-ineligible-click';
+      var ELIGIBLE_VIEW = 'hello-eligible-view';
+      var ELIGIBLE_CLICK = 'hello-open';
+      var START_CONVERSATION = 'hello-start-conversation';
+      var URL_COPIED = 'hello-url-copied';
+      var URL_EMAILED = 'hello-url-emailed';
+      var PAGE_ID_SOURCE = 'hello-tour_OpenPanel_snippet-{{ snippet_id }}';
+
+      (function() {
+          var snippet = document.getElementById('{{ snippet_id }}-hello-simple');
+          snippet.addEventListener('show_snippet', function() {
+              TARGET_AVAILABLE = Mozilla.UITour.isTargetAvailable(TARGET_NAME, true);
+          }, false);
+      })();
+
+      // create namespace
+      if (typeof Mozilla == 'undefined') {
+        var Mozilla = {};
+      }
+
+      ;(function($) {
```

Comment:
Also also you can just re-use the closure above instead of creating a second one.